servicetest/dbus/test_dbus.py

```diff
@@ -98,7 +98,6 @@ def test_query_in(self):
             finally:
                 ca.terminate()
 
-    @pytest.mark.skip(reason="See reported issue about this")
```

Comment:
The issue is in the internal bug tracker though, so it makes little sense to reference it here imho.